cypress/integration/morecast-page.spec.ts

```diff
@@ -13,55 +13,68 @@ describe('MoreCast Page', () => {
+    cy.checkErrorMessage('Error occurred (while fetching GDPS).')
+    cy.checkErrorMessage('Error occurred (while fetching GDPS summaries).')
     cy.checkErrorMessage('Error occurred (while fetching noon forecasts).')
     cy.checkErrorMessage('Error occurred (while fetching noon forecast summaries).')
-    cy.checkErrorMessage('Error occurred (while fetching bias-adjusted models).')
+    cy.checkErrorMessage('Error occurred (while fetching HRDPS).')
+    cy.checkErrorMessage('Error occurred (while fetching HRDPS summaries).')
 
     cy.contains('Data is not available.')
   })
 
   it('if all the weather data were successfully fetched', () => {
-    cy.route('POST', 'api/hourlies/', 'fixture:weather-data/observed-actuals')
-    cy.route('POST', 'api/models/GDPS/predictions/', 'fixture:weather-data/future-models')
-    cy.route('POST', 'api/models/GDPS/predictions/historic/most_recent/', 'fixture:weather-data/past-most-recent-models') // prettier-ignore
-    cy.route('POST', 'api/models/GDPS/predictions/most_recent', 'fixture:weather-data/past-most-recent-models-with-biased-adjusted') // prettier-ignore
-    cy.route('POST', 'api/models/GDPS/predictions/summaries/', 'fixture:weather-data/past-model-summaries')
-    cy.route('POST', 'api/noon_forecasts/', 'fixture:weather-data/forecasts')
-    cy.route('POST', 'api/noon_forecasts/summaries/', 'fixture:weather-data/past-forecast-variations')
-
+    cy.route('POST', 'api/hourlies/', 'fixture:weather-data/actual-readings')
+    cy.route('POST', 'api/noon_forecasts/', 'fixture:weather-data/noon-forecasts')
+    cy.route('POST', 'api/noon_forecasts/summaries/', 'fixture:weather-data/noon-forecast-summaries')
+    cy.route('POST', 'api/models/GDPS/predictions/most_recent', 'fixture:weather-data/models-with-bias-adjusted') // prettier-ignore
+    cy.route('POST', 'api/models/GDPS/predictions/summaries/', 'fixture:weather-data/model-summaries')
+    cy.route('POST', 'api/models/HRDPS/predictions/most_recent', 'fixture:weather-data/hr-models-with-bias-adjusted') // prettier-ignore
+    cy.route('POST', 'api/models/HRDPS/predictions/summaries', 'fixture:weather-data/high-res-model-summaries') // prettier-ignore
     cy.wait('@getStations')
 
     // Request the weather data
     cy.selectStationByCode(stationCode)
     cy.getByTestId('get-wx-data-button').click({ force: true })
 
     // Check if svg elements are displayed in the graph
-    cy.getByTestId('model-summary-temp-area')
     cy.getByTestId('hourly-reading-temp-dot')
-    cy.getByTestId('historic-model-temp-dot')
-    cy.getByTestId('forecast-temp-dot')
-    cy.getByTestId('forecast-summary-temp-line')
+    cy.getByTestId('hourly-reading-rh-dot')
 
     // Test the toggle buttons
     cy.getByTestId('wx-graph-model-toggle').click()
+    cy.getByTestId('model-summary-temp-area')
     cy.getByTestId('model-temp-dot')
     cy.getByTestId('wx-graph-model-toggle').click()
+    cy.getByTestId('model-summary-temp-area').should('not.exist')
```

Comment:
I'm probably just not reading this right, but why do you getByTestId 'model-summary-temp-area' in line 47, and then check that it doesn't exist in line 50?

Reply:
By using `cy.getByTestId` you check if the element is there. Then after I turn off the toggle switch, I make sure that element is no longer shown

Reply:
bah clearly I didn't read line 49. 🤦‍♀️ I think Jake and Sy's brain fogginess has spread to me